### PR TITLE
Make buttons use the accent color

### PR DESCRIPTION
**Pull request by hututudor**
https://etvas.atlassian.net/browse/ETV-4298

* modify the branding provider to make variants for the accent color
* use the accent color in button variants

### CODE REVIEW

File: src/providers/BrandingService.js [hunk 1]
```diff
@@ -19,14 +19,22 @@ const varMapping = {
   brandImage: 'brand-image-url'
 }
 
-const colorVariants = {
+const brandColorVariations = {
```
Comment (alsofronie) [inline src/providers/BrandingService.js:22]: Please use `colorVariations` as single variation object and use it in both brand and accent

File: src/providers/BrandingService.js [hunk 2]
```diff
@@ -51,14 +59,29 @@ export class BrandingService extends EventEmitter {
...
       : {}
 
     this.cssVars = {
       ...newVars,
-      ...brandColorVariants
+      ...brandColorVariants,
+      ...accentColorVariants
     }
 
+    console.warn('colors', this.cssVars)
```
Comment (alsofronie) [inline src/providers/BrandingService.js:83]: Leftover?

**Review by alsofronie — CHANGES_REQUESTED** [review]

File: src/providers/BrandingService.js (hunk 2 above)
Comment (hututudor) [reply src/providers/BrandingService.js:83]: Yes

File: src/providers/BrandingService.js [hunk 3]
```diff
@@ -12,19 +12,29 @@ const varMapping = {
...
   brandColorLighter: 45,
   brandColorLight: 20,
   brandColorDark: -33,
-  brandColorDarker: -66
+  brandColorDarker: -66,
+  accentColorLightest: 70,
+  accentColorLighter: 45,
+  accentColorLight: 20,
+  accentColorDark: -33,
+  accentColorDarker: -66
```
Comment (alsofronie) [inline src/providers/BrandingService.js:37]:
```suggestion
  colorDarker: -66,
  colorLightest: 70,
  colorLighter: 45,
  colorLight: 20,
  colorDark: -33
```

File: src/providers/BrandingService.js [hunk 4]
```diff
@@ -12,19 +12,24 @@ const varMapping = {
...
-  brandColorLighter: 45,
-  brandColorLight: 20,
-  brandColorDark: -33,
-  brandColorDarker: -66
+const colorVariations = {
+  colorLightest: 70,
+  colorLighter: 45,
+  colorLight: 20,
+  colorDark: -33,
+  colorDarker: -66
```
Comment (alsofronie) [inline src/providers/BrandingService.js:32]:
```suggestion
const colorVariations = {
  ColorLightest: 70,
  ColorLighter: 45,
  ColorLight: 20,
  ColorDark: -33,
  ColorDarker: -66
```

File: src/providers/BrandingService.js [hunk 5]
```diff
@@ -51,12 +56,17 @@ export class BrandingService extends EventEmitter {
     const newVars = mergeDeep({}, this.cssVars, updates)
 
     const brandColorVariants = updates.brandColor
-      ? this.buildBrandColorVariants(updates.brandColor, updates)
+      ? this.buildColorVariants({ brandColor: updates.brandColor }, updates)
```
Comment (alsofronie) [inline src/providers/BrandingService.js:59]:
```suggestion
      ? this.buildColorVariants('brand', { brandColor: updates.brandColor }, updates)
```

File: src/providers/BrandingService.js [hunk 6]
```diff
@@ -51,12 +56,17 @@ export class BrandingService extends EventEmitter {
...
+      : {}
+
+    const accentColorVariants = updates.accentColor
+      ? this.buildColorVariants({ accentColor: updates.accentColor }, updates)
```
Comment (alsofronie) [inline src/providers/BrandingService.js:63]:
```suggestion
      ? this.buildColorVariants('accent', { accentColor: updates.accentColor }, updates)
```

File: src/providers/BrandingService.js [hunk 7]
```diff
@@ -100,13 +110,24 @@ export class BrandingService extends EventEmitter {
...
+            colors[computeKeyName(colorName, key)] = shading(
+              colorArray[colorName],
+              colorVariations[key]
+            )
+          }
+          return colors
+        }, {})
+      }),
+      {}
+    )
```
Comment (alsofronie) [inline src/providers/BrandingService.js:130]:
```suggestion
  const buildColorVariants = (prefix, source, existing) =>
    Object.keys(colorVariations).reduce((colors, key) => ({
      ...colors,
      ...(existing[`${prefix}${key}`]
        ? {}
        : { [`${prefix}${key}`]: shading(src, colorVariations[key]) }
      )
    })
```

**Review by alsofronie — CHANGES_REQUESTED** [review]
Please test my code, it is written in github

**Review by alsofronie — APPROVED** [review]

**Pull request merged**